betty/error.py

```diff
@@ -50,5 +49,9 @@ def __reduce__(self) -> tuple[type[Self], tuple[Localizable]]:
     def __str__(self) -> str:
         return self.localize(DEFAULT_LOCALIZER)
 
+    @property
```

Comment:
Do we really need this?  The error itself can be used instead of the contained message because they both implement `Localizable`